### PR TITLE
changelog for 0.72.0 release

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+### 💡 Enhancements 💡
+- Add discoverybundler, initial embedded bundle.d, and enabled properties (#2601)
+- Changes to the pulsarexporter configuration to prepare with using upstream (#2650)
+- Correct module names for directory locations in examples (#2665)
+- Build linux and windows amd64 agent bundles (#2649)
+- Third party dependency updates.
+
+### 🧰 Bug fixes 🧰
+- Add k8sattributes to components.md (#2636)
+- Remove the k8s_tagger from components list (#2638)
```
Comment (dmitryax) [inline CHANGELOG.md:18]: Do we need to count doc updates as bug fixes? I would suggest not to mention them in the changelog

Comment (atoulme) [reply CHANGELOG.md]: removed them

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+## v0.72.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.72.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.72.0) and the [opentelemetry-collector-contrib v0.72.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.72.0) releases where appropriate.
+
+### 💡 Enhancements 💡
+- Add discoverybundler, initial embedded bundle.d, and enabled properties (#2601)
+- Changes to the pulsarexporter configuration to prepare with using upstream (#2650)
+- Correct module names for directory locations in examples (#2665)
+- Build linux and windows amd64 agent bundles (#2649)
+- Third party dependency updates.
```
Comment (aunshc) [inline CHANGELOG.md:14]:
```suggestion
- [Added discoverybundler, initial embedded bundle.d and enabled properties for discovery mode](https://github.com/signalfx/splunk-otel-collector/pull/2601)
- [Updated pulsarexporter configuration to prepare for using exporter from contrib](https://github.com/signalfx/splunk-otel-collector/pull/2650) 
- [Corrected module names for directory locations in examples](https://github.com/signalfx/splunk-otel-collector/pull/2665)
- Built linux and windows amd64 agent bundles (#2649)
- Third party dependency updates
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
 
+## v0.72.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.72.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.72.0) and the [opentelemetry-collector-contrib v0.72.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.72.0) releases where appropriate.
+
+### 💡 Enhancements 💡
+- [Added discoverybundler, initial embedded bundle.d and enabled properties for discovery mode](https://github.com/signalfx/splunk-otel-collector/pull/2601)
+- [Updated pulsarexporter configuration to prepare for using exporter from contrib](https://github.com/signalfx/splunk-otel-collector/pull/2650) 
+- [Corrected module names for directory locations in examples](https://github.com/signalfx/splunk-otel-collector/pull/2665)
+- Built linux and windows amd64 agent bundles (#2649)
```
Comment (atoulme) [inline CHANGELOG.md:13]:
```suggestion
- Built linux and windows amd64 agent bundles (https://github.com/signalfx/splunk-otel-collector/pull/2649)
```

**Pull request merged**